### DISCUSSION TITLE
✅ 🔗 Button icons not aligning properly (Duplicate of #16511)

### DISCUSSION THREAD

**Discussion by akshayurankar48** [elementor/elementor#28080] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

After the elementor recent updates checked the class for modal pop's button and it is coming from elementor , an order of 5 is being added in the said class which is making it appear after even when you set it to before

### Describe the solution you'd like.

Checked  by installing the previous version of Elementor, seems to be working fine until 3.21.8, they've tweaked some things in 3.22.0 from where the issue has started appearing

### Additional context

Just noticed it works fine when you add a cta inside infoBox widget in uae  and if that cta is a text , only for buttons it seems to be breaking , so we'll have to check this further but the issue is from elementor

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there @akshayurankar48!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.28.0! 🥳 

Now all button icons across different Elementor elements are correctly aligned vertically! 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
